src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  peoples: Person[];
+  selectedPersonSlug: string;
+}
+
+export const PeopleTable: FC<Props> = ({
+  peoples,
+  selectedPersonSlug,
+}) => {
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {peoples.length === 0 ? (
+          <p data-cy="noPeopleMessage">
+            There are no people on the server
+          </p>
+        ) : (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Sex</th>
+                <th>Born</th>
+                <th>Died</th>
+                <th>Mother</th>
+                <th>Father</th>
+              </tr>
+            </thead>
+
+            <tbody>
+              {peoples.map(person => {
+                const {
+                  sex,
+                  born,
+                  died,
+                  fatherName,
+                  motherName,
+                  slug,
+                  mother,
+                  father,
+                } = person;
+
+                return (
+                  <tr
+                    data-cy="person"
+                    className={cn({
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.